orm/update.go

```diff
@@ -222,20 +222,27 @@ func (q *updateQuery) appendSetSlice(b []byte) ([]byte, error) {
 		fields = q.q.model.Table().DataFields
 	}
 
+	var table *Table
+	if q.omitZero {
+		table = q.q.model.Table()
+	}
+
 	for i, f := range fields {
 		if i > 0 {
 			b = append(b, ", "...)
 		}
 
 		b = append(b, f.Column...)
 		b = append(b, " = "...)
-		if q.omitZero {
+		if q.omitZero && table != nil {
 			b = append(b, "COALESCE("...)
 		}
 		b = append(b, "_data."...)
 		b = append(b, f.Column...)
-		if q.omitZero {
+		if q.omitZero && table != nil {
 			b = append(b, ", "...)
+			b = append(b, table.Alias...)
```

Comment:
Can you please use:

```
if table.Alias != table.FullName {
    b = append(b, table.Alias...)
    b = append(b, '.')
}
```

Because on some Postgres version alias is not supported.